list/lib/listitem/_list-item.scss

```diff
@@ -73,11 +73,11 @@
     // hide android tap color since we have ripple
     -webkit-tap-highlight-color: transparent;
 
-    &:not(.disabled):not(.noninteractive) {
+    &:not(.disabled) {
       cursor: pointer;
     }
 
-    &.disabled {
+    &.disabled, &.noninteractive {
```

Comment:
What is the use case for disabling pointer events when non-interactive? I would like to still give users the option to add pointer event handlers for the own interactions, especially since you could just set `pointer-events: none` on the host with CSS if that's the desired effect

Reply:
@asyncLiz It's all @vdegenne his fault 👀 
No, i think, if it's non-interactive it makes sense it isn't clickable because you wouldn't be shown a ripple when its clicked plus I think it also shouldn't be intractable with the keyboard

Reply:
I think there are use cases for a ripple that does not have a ripple/focus-ring but can still allow users to listen to click events and perform actions.

At the very least, allowing pointer events gives that option, which an easy workaround to disable them entirely via CSS for those who need them prevented.

Reply:
Okido, I'll force push again :p